src/pages/Home.tsx

```diff
@@ -1,11 +1,184 @@
+  },
+  submitNeed: {
+    width: "45vw",
+    height: "22vh",
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "center",
+    textAlign: "center",
+    minWidth: "210px",
+    minHeight: "185px",
+    marginTop: "10px",
+    background: `linear-gradient(rgba(255, 255, 255, .7), rgba(255, 255, 255, .7)), url(${threePeopleImage}) no-repeat center center `,
+    backgroundSize: "cover",
+    color: "#5b5b5b",
+    fontSize: "17px"
+  },
+  submitLead: {
+    width: "45vw",
+    height: "22vh",
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "center",
+    textAlign: "center",
+    minWidth: "210px",
+    minHeight: "185px",
+    marginTop: "10px",
+    background: `linear-gradient(rgba(255, 255, 255, .7), rgba(255, 255, 255, .7)), url(${maskImage}) no-repeat center center `,
+    backgroundSize: "cover",
+    color: "#5b5b5b",
+    fontSize: "17px"
+  },
+  text: {
+    marginBottom: "15px",
+    fontWeight: 600
+  },
+  textHead: {
+    color: "white",
+    textAlign: "center",
+    width: "30vw",
+    fontWeight: 500,
+    fontSize: "16px",
+    lineHeight: "26px",
+    minWidth: "140px"
+  },
+  submit: {
+    textAlign: "center",
+    bottom: theme.spacing(5)
+  },
+  button: {
+    fontWeight: 400
+  },
+  title: {
+    marginTop: theme.spacing(2),
+    marginLeft: theme.spacing(3),
+    marginBottom: theme.spacing(1)
+  },
+  searchTool: {
+    marginBottom: theme.spacing(1)
+  },
+  modal: {
+    outline: "none"
+  }
+}));
 
 function Home() {
+  const styles = useStyles();
+  const [showNeedSubmitModal, setShowNeedSubmitModal] = useState(false);
+  const [showLeadSubmitModal, setShowLeadSubmitModal] = useState(false);
+  const [error, setError] = useState("");
+  const [mutateNeed] = useMutation(MUTATE_NEED);
+
+  const onNeedSubmit = (need: Need) => {
+    mutateNeed({
+      variables: { request: need }
+    })
+      .then(() => setShowNeedSubmitModal(false))
+      .catch(() => setError("Internal server error, please try again later"));
+  };
+
+  const onLeadSubmit = (lead: Lead) => {
+    mutateNeed({
+      variables: { request: lead }
+    })
+      .then(() => setShowLeadSubmitModal(false))
+      .catch(() => setError("Internal server error, please try again later"));
+  };
+
   return (
     <>
       <NavBar />
-      <h1>Home page</h1>
+      <Box className={styles.head}>
+        <Container className={styles.textHead}>
+          Aggregating global demand & supply of medical supplies & equipment to
+          connect needs with supply leads. STAT.
+        </Container>
+      </Box>
+      <Box className={styles.container}>
+        <Container className={styles.submitNeed}>
+          <Container className={styles.text}>
+            On the front-lines flighting COVID-19 and in need of medical
+            supplies & equipment?
+          </Container>
+          <Container className={styles.submit}>
+            <Button
+              onClick={() => setShowNeedSubmitModal(true)}
```

Comment:
User needs to be authenticated to open this modal

Reply:
Talked with you about this on Slack, this will be implemented in a different ticket